packages/@storefront/flux-capacitor/src/core/configuration.ts

```diff
@@ -214,6 +214,10 @@ namespace Configuration {
      * enable updating query when hovering over sayt elements
      */
     hoverAutoFill?: boolean;
+    /**
+     * specify optional debounce value in milliseconds
+     */
+    debounce?: number;
```

Comment:
This name signals a boolean value to me. Maybe call it `debounceDelay` or `debounceThreshold`. Remember to change the changelog.

Reply:
I was under the impression that we follow the practice of prefixing our boolean values with `is/has/should/etc.`? In any case, I think your suggestions *are* better than just `debounce`. Will update.

Reply:
Changed to `debounceThreshold `. Resolving.